apps/crn-frontend/src/network/__tests__/Network.test.tsx

```diff
@@ -162,6 +164,30 @@ it('allows selection of filters', async () => {
   );
 });
 
+it('allows selection of group filters', async () => {
+  const { getByText, getByLabelText } = await renderNetworkPage(
+    network({}).groups({}).$,
+  );
+
+  userEvent.click(getByText('Filters'));
+  const checkbox = getByLabelText('Active');
+  expect(checkbox).not.toBeChecked();
+
+  userEvent.click(checkbox);
+  expect(checkbox).toBeChecked();
+  await waitFor(() =>
+    expect(mockGetGroups).toHaveBeenLastCalledWith(
+      {
+        currentPage: 0,
+        filters: new Set(['Active']),
+        pageSize: 10,
+        searchQuery: '',
+      },
```

Comment:
consider using expect.ObjectMatching to make your tests more robust if we add things to requests
```suggestion
      expect.objectMatching({
        filters: new Set(['Active']),
      },
```